include/boost/sml/aux_/utility.hpp

```diff
@@ -152,7 +152,8 @@ struct pool_type_impl : pool_type_base {
 };
 
 template <class T>
-struct pool_type_impl<T &, aux::enable_if_t<aux::is_constructible<T>::value && aux::is_constructible<T, T>::value>> : pool_type_base {
```

Comment:
is that clang.format > 9? ci is tseting against clang-format 9 I think

Reply:
It was clang-format-10. I switched to clang-format-9 and re-ran. Hopefully it's good now.